File: solucoes/2008/nivel jr/fase 2/js/auto.js
var N, E
var paineis = 0
scanf("%d%s", "N", "E")

for(var i = 0; i < N; i++){
    if(E[i] == 'P'){
        paineis += 2
    }
    else if (E[i] == 'C'){
        paineis += 2
    }
    else if(E[i] == 'A'){
        paineis += 1
    }
}
printf("%d", paineis)

File: solucoes/2008/nivel jr/fase 2/py/auto.py
N = int(input())
est = str(input())
paineis = 0
for i in range(N):
    if(est[i] == 'P'):
        paineis += 2
    elif est[i] == 'C':
        paineis += 2
    elif est[i] == 'A':
        paineis += 1
print(paineis)
